src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vilkårsvurdering/forskyvVilkår.kt

```diff
@@ -115,15 +115,18 @@ fun forskyvVilkårResultater(
         }.filter { (it.fom ?: TIDENES_MORGEN).isBefore(it.tom ?: TIDENES_ENDE) }
 }
 
-private fun tilVilkårResultaterMedInformasjonOmNestePeriode(vilkårResultater: List<VilkårResultat>) =
-    vilkårResultater.zipWithNext { denne, neste ->
+private fun tilVilkårResultaterMedInformasjonOmNestePeriode(vilkårResultater: List<VilkårResultat>): List<VilkårResultaterMedInformasjonOmNestePeriode> {
+    if (vilkårResultater.isEmpty()) return emptyList()
```

Comment:
Hvorfor trenger vi dette? 
https://pl.kotl.in/E0KbjOKTR

Reply:
ah. Det er jo for når vi bruker `.last()`

Reply:
Stemmer, vi bruker last på slutten😛